src/OVAL/probes/unix/file.c

```diff
@@ -272,17 +274,25 @@ static int file_cb (const char *p, const char *f, void *ptr)
 	if (f == NULL) {
 		st_path = p;
 	} else {
-		snprintf (path_buffer, sizeof path_buffer, "%s/%s", p, f);
+		const size_t p_len = strlen(p);
+		/* Avoid 2 slashes */
+		if (p_len >= 1 && p[p_len - 1] == FILE_SEPARATOR) {
+			snprintf(path_buffer, sizeof path_buffer, "%s%s", p, f);
+		} else {
+			snprintf(path_buffer, sizeof path_buffer, "%s/%s", p, f);
```

Comment:
Use FILE_SEPARATOR instead of hardcoded /

Reply:
OK, but It was there

Reply:
IMO just make it consistent. I don't care if it's changed here or in another PR. I would change it here to save effort.